pkg/policy/api/l4.go

```diff
@@ -150,6 +150,19 @@ type Listener struct {
 	// +kubebuilder:validation:MinLength=1
 	// +kubebuilder:validation:Required
 	Name string `json:"name"`
+
+	// Priority is used to select the Listener if multiple rules would apply different listeners
+	// to a policy map entry. Lower numbers indicate higher priority. If left out, a random, but
```

Comment:
I think there's one thing that this description doesn't super explicitly point out today, which could be clearer: The traffic will only be redirected to one listener. The priority aspect is super clear but the 'control flow' is not as obvious to me.

Reply:
Given the implementation dependent nature of this definition I heeded your overall concern and changed the API description to:
```
	// Priority for this Listener that is used when multiple rules would apply different
	// listeners to a policy map entry. Behavior of this is implementation dependent.
```